doc/cpd/101.rst

```diff
@@ -75,6 +78,16 @@ if set to ``True`` would use the secrets handler.
 The secrets handler would be automatically added to the ``StorageHandlerManager`` and
 would require no special configuration by the operator.
 
+New HTTP Handler
+~~~~~~~~~~~~~~~~
+A new handler called ``secrets`` could be added. This would proxy requests back to the
+Custodia instance.
```

Comment:
Got'cha.  So in that case do we still need the `"custodia_api_id"` and `"custodia_api_key"` things in the storage config?  It's not clear to me what those are for.